frontend/controller/actions/group.js

```diff
@@ -316,7 +316,9 @@ export default (sbp('sbp/selectors/register', {
   // action if we haven't done so yet (because we were previously waiting for
   // the keys), or (c) already a member and ready to interact with the group.
   'gi.actions/group/join': async function (params: $Exact<ChelKeyRequestParams> & { options?: { skipUsableKeysCheck?: boolean; skipInviteAccept?: boolean } }) {
-    sbp('okTurtles.data/set', 'JOINING_GROUP-' + params.contractID, true)
+    if (!params.options?.skipInviteAccept) {
+      sbp('okTurtles.data/set', 'JOINING_GROUP-' + params.contractID, true)
```

Comment:
I need to test this, but I'd think it breaks waiting for an invite upon logging in (i.e., logging in before the keys have been received).

Reply:
`'JOINING_GROUP-${contractID}'` is being used inside the sideEffect functions of group contracts.
It should be true only when joining group, not when try to log in or to see if the user has key.
That's why I added `if` conditional statement.